tests/phpunit/tests/dependencies/wpScriptTag.php

```diff
@@ -8,7 +8,7 @@
  */
 class Tests_Functions_wpScriptTag extends WP_UnitTestCase {
 
-	function get_script_tag_type_set() {
+	public function get_script_tag_type_set() {
```

Comment:
Hmm, this test needs a `test_` in front of it for it run. Outside the scope of this ticket. But it needs to be fixed.

Reply:
👍  Also needs:

1. A `$message` value for the two assertions or, ideally, a data provider.
2. A `@covers` tag for `::wp_get_script_tag`.
3. Some work on the test as both assertions currently fail for the following reasons:

Test #1 - `wp_get_script_tag()` returns the attributes in the order they were supplied. The expected value has the attributes in a different order than it supplied to `wp_get_script_tag()`.

Test #2 - `wp_get_script_tag()` calls `wp_sanitize_script_attributes()`. For boolean attributes, this returns `attribute="attribute"`. The test supplies `'nomodule' => true` and tests for `nomodule`, but gets `nomodule="nomodule"` instead. IMO, this should be part of a wider discussion in Core about the usage of `attribute="attribute"`. However, that is beyond the scope of fixing that test, which just needs some minor corrections in the assertions.